app/controllers/persons_controller.rb

```diff
@@ -8,11 +8,12 @@ class PersonsController < ApplicationController
 
   def index
     @persons = get_persons
+    json_people = Person.all
     @fcn_link = Webpage.find_by(title: "Frequently called numbers")
 
     respond_to do |format|
       format.html
-      format.json { render json: PersonSerializer.new(@persons.to_a) }
+      format.json { render json: PersonSerializer.new(json_people.to_a) }
```

Comment:
@cdoyle-temple I think it would be more efficient to not set `json_people = Person.all` and instead just do 
```
format.json { render json: PersonSerializer.new(json_people.to_a) }
```
Reason being that currently it loads `Person.all` even in cases where you do just want to display a limit of 20 (i.e. format.html).